client/setup.sh

```diff
@@ -1,9 +1,9 @@
-dnf install -y git-core autoconf automake pkg-config python36 wget binutils make zip
+dnf install -y git-core autoconf automake pkg-config python38 wget binutils make zip
```

Comment:
this is technically not needed as part of the upgrade since it's just the test client, but probably good to do anyway